bindings/python/iota_sdk/utils.py

```diff
@@ -43,7 +44,7 @@ def nft_id_to_bech32(nft_id, bech32_hrp):
         })
 
     @staticmethod
-    def hex_public_key_to_bech32_address(hex, bech32_hrp=None):
+    def hex_public_key_to_bech32_address(hex: str, bech32_hrp=None) -> str:
```

Comment:
Is there no type int when there is also a default value?

Reply:
There is, would be `Optional[str]`, but was actually wrong since it's not optional here
Updated it